code/code/sys/gmcphandlers.cc

```diff
@@ -0,0 +1,138 @@
+#include "gmcphandlers.h"
+#include "being.h"
```

Comment:
code/sys/gmcphandlers.cc:6:20: fatal error: json.hpp: No such file or directory
 #include "json.hpp"

On docker in windows 10.